Gopkg.lock

```diff
@@ -36,39 +34,31 @@
     "private/protocol/restxml",
     "private/protocol/xml/xmlutil",
     "service/s3",
-    "service/sts",
+    "service/sts"
```

Comment:
Which version of `dep` are you using btw? I'm on `v0.5.0` and it adds a comma for me here. Woul

Reply:
```
dep version
dep:
 version     : devel
 build date  :
 git hash    :
 go version  : go1.10
 go compiler : gc
 platform    : darwin/amd64
 features    : ImportDuringSolve=false
```